lib/steep/typing.rb

```diff
@@ -41,7 +41,7 @@ def initialize(source:, root_context:, parent: nil, parent_last_update: parent&.
     end
 
     def add_error(error)
-      errors << error
+      errors << error unless source.directive_map.ignored?(error)
```

Comment:
I need to know where is the best place to suppress the errors by directive comments.
Is there better one?